docs/calamari/node-operators/introduction.md

```diff
@@ -6,4 +6,13 @@ sidebar_position: 1
 
 ## What is Calamari parachain
 
-### What is a shell-chain 
\ No newline at end of file
+Calamari, Manta Network’s canary-net (the canary-net is an early, highly experimental version of Manta Network, where all new features will be released and tested first), is the plug-and-play privacy-preservation parachain built to service the Kusama DeFi world. It combines Kusama and zkSNARKs to bring on-chain privacy to transactions and swaps. As part of its own suite of products, Calamari Network offers:
+
+In early period, Calamari parachain will provide two products:
+* **MariPay**, its token-agnostic private payment service. MariPay supports the private transfer of Kusama and its Parachain assets, including major crypto assets supported on Kusama. Users can transact popular assets like stablecoins and wrapped BTC while simultaneously reaping the benefits of on-chain privacy through ZKP.
+* **MariSwap**, its private AMM-based DEX. MariSwap offers users the capability of swaps between parachain assets while preserving the privacy of the user addresses. It also provides never-before-seen features in the industry such as private liquidity pools.
+
+Calamari will be the only platform on Kusama that can meet user on-chain privacy needs, boost anti-surveillance, and ensure private tokens’ decentralization and interoperability. Now Calamari we provide is a shell-chain.
+
```

Comment:
``Currently Calamari is a shell-chain`` I think we should start with that.

Reply:
Yeah, I agree. But I am not very sure how to explain to shell chain